go.mod

```diff
@@ -51,7 +49,7 @@ require (
 	github.com/cinience/go_rocketmq v0.0.2
 	github.com/cloudevents/sdk-go/binding/format/protobuf/v2 v2.14.0
 	github.com/cloudevents/sdk-go/v2 v2.14.0
-	github.com/cloudwego/kitex v0.5.0
+	github.com/cloudwego/kitex v0.9.0
```

Comment:
Is there anything we can do to confirm this works? Between 0.5.0 and 0.9.0 there has been [1 year worth of releases](https://github.com/cloudwego/kitex/releases). I don't believe we have automated tests for this today, so is there anything we can do to validate it works?